src/components/Jobs/RunningJobs/RunningJobs.js

```diff
@@ -1,33 +1,22 @@
 import React, { useContext } from 'react';
-import { isEmpty } from 'lodash';
 
 import {
   sortRunningJobs,
   JobsListAccordion,
 } from '@folio/stripes-data-transfer-components';
 
-import { JOB_EXECUTION_STATUSES } from '../../../utils/constants';
 import ItemFormatter from './ItemFormatter';
 import { DataFetcherContext } from '../../../contexts/DataFetcherContext';
-import { sampleJobExecution } from '../../../../test/bigtest/network/scenarios/fetch-job-profiles-success';
-
-const prepareJobsData = jobs => {
```

Comment:
Do we really need to remove this filter? It seems, we will support scheduled jobs in the future, for example.

Reply:
Then I suggest adding it in the future. Moreover, I envision another server call for queued entries instead of combining both in one request as fetching them both can contain only lists of running jobs without queued.

Reply:
It seems, we will have at least 4 calls to backend every 5 seconds to retrieve the same data but with different status: running jobs, jobs logs, scheduled jobs, queued jobs. Is it ok?

Reply:
I suggest discussing it with the backend team. This is how I see this should be for now. Another case is to set a high limit and then possibly all items will be fetched, but in this way, a lot of data will be fetched every five seconds as opposed to multiple calls but for less data.

Reply:
Okay. Let's discuss it in Q2.